src/views/App/index.tsx

```diff
@@ -72,6 +73,15 @@ class App extends React.Component<MainProps, MainState> {
                     component={GovernanceTaskList}
                   />
                 )}
+
+                {['local', 'dev', 'impl'].includes(
```

Comment:
There's no need to hide this route in prod. We'll eventually put it behind a check for user roles, but for now there's no leaked data, so we can just let it be.